File: mobile/src/main/java/treehou/se/habit/ui/servers/serverlist/ServersPresenter.java
package treehou.se.habit.ui.servers.serverlist;

import javax.inject.Inject;

import treehou.se.habit.module.RxPresenter;
import treehou.se.habit.ui.settings.SettingsContract;

public class ServersPresenter extends RxPresenter implements ServersContract.Presenter {

    private ServersContract.View view;

    @Inject
    public ServersPresenter(ServersContract.View view) {
        this.view = view;
    }
}

File: mobile/src/main/java/treehou/se/habit/ui/servers/serverlist/ServersPresenter.kt
package treehou.se.habit.ui.servers.serverlist

import android.os.Bundle
import javax.inject.Inject

import treehou.se.habit.dagger.RxPresenter

class ServersPresenter @Inject
constructor(private val view: ServersContract.View) : RxPresenter(), ServersContract.Presenter {

    override fun load(launchData: Bundle?, savedData: Bundle?) {
        super.load(launchData, savedData)


    }
}
